src/main/java/general/ui/Text.java

```diff
@@ -0,0 +1,15 @@
+package general.ui;
+
+import java.util.stream.Stream;
+
+public interface Text {
```

Comment:
I like your class Text to process String in the UI.

Reply:
Thanks! I created the interface mainly because I had envisioned for proper rich text support, so whatever that is passed in may not be a pure `String`, so I wanted to make a common standard `Text` that was understandable by the app as something to parse and print accordingly